gulpfile.js

```diff
@@ -0,0 +1,44 @@
+var gulp                = require('gulp'),
+    livereload          = require('gulp-livereload'),
+    nodemon             = require('gulp-nodemon'),
+    shell               = require('gulp-shell'),
+    chalk               = require('chalk'),
+    paths,
+    nodemonServerInit;
+
+paths = {
+    server: 'core/server/',
+    client: 'core/client/'
+};
+
+nodemonServerInit = function () {
+    livereload.listen();
+
+    return nodemon({
+        script: 'index.js',
+        ext: 'js,json',
+        watch: [
+            'core/index.js',
+            paths.server + '**/*.js',
+            'core/built/assets/*.js'
+        ],
+        ignore: [
+            'core/client/*',
+            'core/server/test/*'
+        ]
+    }).on('restart', function () {
+        console.info(chalk.cyan('Restarting Ghost due to changes...'));
+        gulp.src('index.js')
+            .pipe(livereload());
+    });
+};
+
+// Starting Ember for development
+gulp.task('admin', shell.task(['cd ' + paths.client + ' && ember build --watch']));
+
+gulp.task('serve', function () {
```

Comment:
I meant that the `gulp admin` task would need to pass options through, only time you'd want to run `ember server` by itself is when you are passing in custom options.

I'm also not really convinced by the benefits of disabling the watching in either case? Does `nodemon` or `watchman` really use much resources? My understanding of how it works is that they rely on OS-level events for performance so essentially do nothing/use next to no resources unless files are actually changed.